### PR TITLE
Refactor: Create new packet instances instead of reusing the old ones (issue #179)

### PR DESCRIPTION
**Pull request by rafin**
> Create new packet instances instead of reusing the old ones  Issue #179 

All packets (except `TCState()` in `player.py`) refactored to be recreated instead of reused upon sending. Minimal testing.

### CODE REVIEW

File: pyspades/server.py [hunk 1]
```diff
@@ -57,7 +54,7 @@ class ServerProtocol(BaseProtocol):
     connections = None
     player_ids = None
     master = False
-    max_score = 10
+    max_score = 1
```
Comment (swalladge) [inline pyspades/server.py]: Was this an accidental change? :)

Comment (rafin) [reply pyspades/server.py]: Yes, whoops.

**Comment by NotAFile** [comment]
LGTM if you fix the typo thing.

It would be great if you made the individual commits more substantial in the future, making them this small generates a lot of noise in the logs. The solution would be to squash merge, but since there are many authors the author information would get lost and I don't really want that.

**Comment by rafin** [comment]
We fixed the typo. There was also another packet initialization that we missed - should be good now.

**Comment by NotAFile** [comment]
Have you tested this in-game?

**Comment by rafin** [comment]
Yes, with 3 players connected simultaneously.

**Comment by NotAFile** [comment]
This is starting to become too many commits for the amount of actual changes. I'm afraid we'll have to squash merge this.

**Comment by rafin** [comment]
That makes sense, not an issue for us.

**Comment by NotAFile** [comment]
Is this ready to be merged/tested?

**Comment by rafin** [comment]
Yes it is, I'll fix the title.

**Comment by NotAFile** [comment]
I guess we'll just merge and see if something breaks?

**Pull request merged**